x-pack/plugins/timelines/server/search_strategy/timeline/index.ts

```diff
@@ -129,7 +129,10 @@ const timelineAlertsSearchStrategy = <T extends TimelineFactoryQueryTypes>({
 }) => {
   // Based on what solution alerts you want to see, figures out what corresponding
   // index to query (ex: siem --> .alerts-security.alerts)
-  const indices = alertConsumers.flatMap((consumer) => mapConsumerToIndexName[consumer]);
+  let indices = request.defaultIndex ?? request.indexType;
```

Comment:
this change looks slightly outside the scope of the field name changes, was there a particular reason to include it here?

Reply:
Should these be flipped as well to use the `indexType` and if that's nullish default to the `defaultIndex`?

Reply:
Updated to remove this logic, works by adding `*` to `.alerts-security.alerts`. Though I think what we need to do is update the getSignalsIndex endpoint to return the alias, not just `.siem-signals....`